### DISCUSSION TITLE
'Storybook' style demo element

### DISCUSSION THREAD

**Discussion by braven112** [AlaskaAirlines/WC-Generator#439] [Announcements]
## What is the request? 

The following project is a Storybook-style HTML custom element that we could consider for use with your demo pages. 

https://github.com/web-padawan/api-viewer-element

**Comment by blackfalcon** [comment]
Here is an example of a resizer UI I created to help support an API viewer. 

https://codepen.io/blackfalcon/pen/VwzxoLa
